FiveCalls/FiveCalls/ImpactViewModel.swift

```diff
@@ -9,30 +9,35 @@
 import Foundation
 
 struct ImpactViewModel {
-    let logs: [ContactLog]
     
-    init(logs: [ContactLog]) {
+    let userStats: UserStats?
+    let logs: ContactLogs
+    
+    init(logs: ContactLogs, stats: UserStats?) {
```

Comment:
Just so I have this clear, this is a transitional thing...  we need to keep around ContactLogs for other areas of the app currently, but until we unify the entire thing we have to merge these 2 sets?

Reply:
`ContactLogs` wouldn't be strictly necessary if the server were providing completion status for issues, but it would be useful to keep it so that we can keep track of calls that we weren't able to report to the server due to connectivity issues or server downtime.  In that scenario we would only keep the `ContactLogs` for unreported calls, instead of keeping them all on the device forever the way we do now.  If we got completion data from the server we could wipe the local entry as soon as we were able to report it.  As of now though we definitely still need to keep all of the local stats so we can use them to mark up the issues list properly to show what the user has already done.